src/OpenIddict.Core/Infrastructure/OpenIddictProvider.Authentication.cs

```diff
@@ -67,7 +70,14 @@ public partial class OpenIddictProvider<TUser, TApplication, TAuthorization, TSc
                 }
 
                 // Restore the authorization request parameters from the serialized payload.
-                context.Request.Import(payload);
```

Comment:
FYI, the removed `Import`/`Export` helpers used binary serialization (via `BinaryReader`/`BinaryWriter`). To support complex serialization, we're going to move to BSON.